website/common/script/ops/changeClass.js

```diff
@@ -35,8 +35,6 @@ module.exports = function changeClass (user, req = {}, analytics) {
         }
       }
 
-      user.items.gear.equipped[type] = foundKey;
```

Comment:
Looks like all of this `foundKey` logic can be removed as it's only used for this line.

You'll need to update the corresponding test as well.